README.md

```diff
@@ -38,6 +38,7 @@ When the `avrogo` command generates Go datatypes from Avro schemas, it uses the
   as `time.Time` type
 - `{"type": "string", "logicalType": "uuid"}` is represented as
   [github.com/google/uuid.UUID](https://pkg.go.dev/github.com/google/uuid#UUID) type.
+- `{"type": "long", "name": "duration-nanos"}` is represented as `time.Duration` type.
```

Comment:
Shouldn't it be

`{"type": "long", "logicalType": "duration-nanos"}`

?

Reply:
you're right :)